test/integration/copagame/copagame_test.clj

```diff
@@ -0,0 +1,61 @@
+
+(clojure.test/use-fixtures :once schema.test/validate-schemas)
+
+(defn- create-and-start-components! []
+  (component/start-system
+   (component/system-map
+    :config (components.config/new-config)
+    :http (components.http/new-http-mock {})
+    :router (components.router/new-router routes/routes)
+    :webserver (component/using (components.webserver/new-webserver)
+                                [:config :http :router]))))
+(defflow
+  flow-integration-copagame-test
+  {:init (util/start-system! create-and-start-components!)
+   :cleanup util/stop-system!
+   :fail-fast? true}
+  (flow "should interact with system"
+    (flow "prepare system with http-out mocks"
+      (util.http/set-http-out-responses! {"https://l3-processoseletivo.azurewebsites.net/api/Competidores?copa=games"
+                                          {:body [{:id "/nintendo-64/the-legend-of-zelda-ocarina-of-time"
+                                                   :titulo "The Legend of Zelda: Ocarina of Time (N64)"
+                                                   :nota 99.9
+                                                   :ano 1998,
+                                                   :urlImagem "https://l3-processoseletivo.azurewebsites.net/api/CapaJogo/nintendo-64/the-legend-of-zelda-ocarina-of-time"},
+                                                  {:id "/playstation/tony-hawks-pro-skater-2"
+                                                   :titulo "Tony Hawk's Pro Skater 2 (PS)"
+                                                   :nota 98.9
+                                                   :ano 2000
+                                                   :urlImagem "https://l3-processoseletivo.azurewebsites.net/api/CapaJogo/playstation/tony-hawks-pro-skater-2"}]
+                                           :status 200}})
+
+      (flow "should list wallet deposits"
+        (match? (matchers/embeds {:status 200
+                                  :body [{:id "/nintendo-64/the-legend-of-zelda-ocarina-of-time"
+                                          :titulo "The Legend of Zelda: Ocarina of Time (N64)"
+                                          :nota 99.9
+                                          :ano 1998,
+                                          :urlImagem "https://l3-processoseletivo.azurewebsites.net/api/CapaJogo/nintendo-64/the-legend-of-zelda-ocarina-of-time"},
+                                         {:id "/playstation/tony-hawks-pro-skater-2"
+                                          :titulo "Tony Hawk's Pro Skater 2 (PS)"
+                                          :nota 98.9
+                                          :ano 2000
+                                          :urlImagem "https://l3-processoseletivo.azurewebsites.net/api/CapaJogo/playstation/tony-hawks-pro-skater-2"}]})
+                (util.webserver/request! {:method :get
+                                          :uri    "/copagame/read"}))))))
```

Comment:
I have my doubts whether this test is really worth it. As it stands
1) You mock the endpoint
2) Fire the request
3) Assert the response is the mocked thing.

It doesn't add too much value, but on the other hand it might create a good foundation for the next tests when the features becomes more complex.

Anyway, I think we should keep it. The only change I'd request is to perhaps create a variable for `body` that we can reference in both places to help with readability?
```clj
(def mocked-movies ...)
(util.http/set-http-out-responses! {"https://l3-processoseletivo.azurewebsites.net/api/Competidores?copa=games"
                                          {:body mocked-movies
                                           :status 200}})
...
(matchers/embeds {:status 200 :body mocked-movies} ...
```

Reply:
Nice. Changed! I added def to the teams.